### PR TITLE
Update thrift connector doc for retry configs

### PR DESCRIPTION
**Pull request by hellium01**

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/thrift.rst [hunk 1]
```diff
@@ -42,6 +42,11 @@ Property Name                                 Description
 ``static-location.hosts``                     Location of Thrift servers
 ``presto-thrift.max-response-size``           Maximum size of a response from thrift server
 ``presto-thrift.metadata-refresh-threads``    Number of refresh threads for metadata cache
+``presto-thrift.max-retry-attempts``          Number of retries before finally fails a thrift request
```
Comment (AlekseiS) [inline presto-docs/src/main/sphinx/connector/thrift.rst]: It seems like "..before finally fails.." is not the right working. Maybe, ".. before thrift request finally fails"? @electrum please suggest a better wording.

Comment (hellium01) [reply presto-docs/src/main/sphinx/connector/thrift.rst]: Maybe "Maximum number of retry attempts for failed retryable thrift requests"? borrowed from AWS's document :-) :
http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/ClientConfiguration.html#DEFAULT_RETRY_POLICY

File: presto-docs/src/main/sphinx/connector/thrift.rst [hunk 2]
```diff
@@ -42,6 +42,11 @@ Property Name                                 Description
 ``static-location.hosts``                     Location of Thrift servers
 ``presto-thrift.max-response-size``           Maximum size of a response from thrift server
 ``presto-thrift.metadata-refresh-threads``    Number of refresh threads for metadata cache
+``presto-thrift.max-retry-attempts``          Maxium number of retries for failed retryable thrift request
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/thrift.rst]: "Maximum number of retries for failed Thrift requests"

File: presto-docs/src/main/sphinx/connector/thrift.rst [hunk 3]
```diff
@@ -42,6 +42,11 @@ Property Name                                 Description
...
+``presto-thrift.min-retry-sleep-time``        Min interval between retry attempts
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/thrift.rst]: Minimum

File: presto-docs/src/main/sphinx/connector/thrift.rst [hunk 4]
```diff
@@ -42,6 +42,11 @@ Property Name                                 Description
...
+``presto-thrift.max-retry-sleep-time``        Max interval between retry attempts
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/thrift.rst]: Maximum

File: presto-docs/src/main/sphinx/connector/thrift.rst [hunk 5]
```diff
@@ -42,6 +42,11 @@ Property Name                                 Description
...
+``presto-thrift.min-retry-duration``          Max duration of a retryable thrift request before finally fails it
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/thrift.rst]: "Maximum duration across all attempts of a Thrift request"

**Comment by hellium01** [comment]
Thanks, @electrum , updated the PR for the suggestion. Please let me know if everything looks OK.

**Pull request closed without merging**